### PR TITLE
Support cross kaltura target url manipulation based on worker config

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

File: plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php [hunk 1]
```diff
@@ -142,12 +166,28 @@ protected function initClients(KalturaCrossKalturaDistributionProfile $distribut
...
 	}
+	
+	private function getTargetServiceUrl($serviceUrl)
+	{
+		if(!$this->targetServiceUrlRegex)
+		{
+			return $serviceUrl;
+		}
+		
+		$matches = array();
```
Comment (amiras89) [inline plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php:183]: Do we need it? it is not being used as far as I can see.

**Review by amiras89 — APPROVED** [review]

File: plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php (hunk 1 above)
Comment (yossipapi) [reply plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php]: I was using it before and forgot to remove it.
Removed now.

File: plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php [hunk 2]
```diff
@@ -142,12 +166,27 @@ protected function initClients(KalturaCrossKalturaDistributionProfile $distribut
...
 	}
+	
+	private function getTargetServiceUrl($serviceUrl)
+	{
+		if(!$this->targetServiceUrlRegex)
+		{
+			return $serviceUrl;
+		}
+		
+		if (!preg_match($this->targetServiceUrlRegex, $serviceUrl, $matches))
```
Comment (amiras89) [inline plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php:183]: need to remove the $matches from here too :)

Comment (yossipapi) [reply plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php]: Done.

**Pull request merged**